analysis_notebooks/Geography Analysis.ipynb

```diff
@@ -698,6 +757,26 @@
     "choropleth()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Quantized relationship between (income) and image counts normalized by (population/area size)"
```

Comment:
why is income in parentheses?

Reply:
it was just a notation choice to leave parameters in parenthesis since the user can modify these to their choice. I'll remove for better clarity